### ISSUE TITLE
Build & publish linux/arm64 Docker image

### ISSUE THREAD

**Issue by cap10morgan** [babashka/babashka#1097]
Once #1095 or similar lands, it would be nice to have the CI/CD pipeline use it to build and publish a linux/arm64 Docker image to Docker Hub in addition to the linux/amd64 already there.

Mostly opening this issue because I'm already working on this locally while waiting for #1095's CI infrastructure issues to settle down. :)

I have the [modified actions running in my fork](https://github.com/cap10morgan/babashka/actions) and I'll open a PR here when everything is working.

**Comment by cap10morgan** [comment]
Merging #1095 will not close this issue. This will be covered by a future PR.

**Comment by cap10morgan** [comment]
@borkdude Should this be done in GitHub Actions or CircleCI?

**Comment by lispyclouds** [comment]
@cap10morgan CircleCI is preferred for this.
